lib/ParamedicConfig.js

```diff
@@ -104,4 +117,44 @@ ParamedicConfig.prototype.getTimeout = function() {
     return DEFAULT_TIMEOUT;
 };
 
+ParamedicConfig.prototype.getLogPath = function() {
+    return this._config.logPath;
+};
+
+ParamedicConfig.prototype.setLogPath = function(logPath) {
+    this._config.logPath = logPath;
+};
+
+ParamedicConfig.prototype.getLogMins = function() {
+    return this._config.logMins;
+};
+
+ParamedicConfig.prototype.setLogMins = function(logMins) {
+    this._config.logMins = logMins;
+};
+
+ParamedicConfig.prototype.setAppName = function(appName) {
+    this._config.appName = appName;
+};
+
+ParamedicConfig.prototype.getAppName = function() {
+    return this._config.appName;
+};
+
+ParamedicConfig.prototype.setSimulatorsFolder = function(simulatorsFolder) {
+    this._config.simulatorsFolder = simulatorsFolder;
```

Comment:
Consider using what we do for getting simulator logs. Ideally we should have one way of doing this and not configurable externally.